tests/unit_tests/test_deplete_integrator.py

```diff
@@ -174,6 +192,14 @@ def test_integrator(run_in_tmpdir, scheme):
     assert dep_time.shape == (2, )
     assert all(dep_time > 0)
 
+    integrator = bundle.solver(operator, [0.75], 1, solver=cram.CRAM48)
+    assert integrator.solver is cram.CRAM48
+
+    integrator.solver = mock_good_solver
+    assert integrator.solver is mock_good_solver
+
+    lfunc = lambda A, n, t: mock_good_solver(A, n, t)
+
```

Comment:
Can be removed?

Reply:
I left out a line showing that we could make lambda functions work too, if someone wanted to do that. Trying to add all tests cases. We can leave it out, or add in like a simply cython / non-inspectable function